she/promote.py

```
@@ -185,6 +185,10 @@ def plan_promotion(
     verification = verification or plan_verification(incident)
     if check_results:
         verification = apply_check_results(verification, check_results)
+    if verification.incident_id != incident.incident_id:
+        raise PromotionError("verification incident_id must match incident")
```

Ruff (0.16.3)

[warning] Avoid specifying long messages outside the exception class

(TRY003)